src/lib/services/emissionsCalculator.ts

```diff
@@ -36,12 +36,19 @@ export default class EmissionsCalculator {
     energyConsumedWh: number
   ): Promise<number> {
     const url = this.intensityUrl(new Date(new Date(productionStartDate).getTime() - 1000 * 60 * 60), productionEndDate)
-    const response = await fetch(url)
-    if (response.status !== 200) {
-      throw new InternalServerError('Unexpected error fetching carbon intensity data')
+    const hardcodedFactor = 0.07
```

Comment:
Can we randomise this between 2 reasonable limits

Reply:
Any suggestions ?

Reply:
Added a random factor ( [0.03, 0.11] )